test/data/fhir2cmumps-patient7.shex

```diff
@@ -0,0 +1,580 @@
+﻿PREFIX fhir: <http://hl7.org/fhir/> 
```

Comment:
Does this file need to be checked in?  Or is it auto-generated from cmumps2fhir-patient7.shex?